tests/unit/test_mesos_url.py

```diff
@@ -0,0 +1,16 @@
+# test.py
```

Comment:
You have to make it a pytest test, otherwise is not being executed. Something like this:

```
import mock
import unittest

class MesosTests(unittest.TestCase):
    def test_xxxx(self):
        # assert something
```

https://github.com/cloudviz/agentless-system-crawler/blob/master/tests/unit/test_container.py